src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -29,8 +29,14 @@ public class Command
 
         private bool _running = false;
 
-        private Command(string executable, string args)
+        //the command is not loaded from nuget libraries or from AppContext.BaseDirectory. It is machine specific.
+        //see https://github.com/dotnet/cli/issues/382
+        private readonly bool _isUnsafe;
+
+        private Command(string executable, string args, bool isUnsafe)
         {
+            _isUnsafe = isUnsafe;
```

Comment:
Initially I had called the flag `loadedFromPath` :)
However, that name will become misleading in the future

Right now we accept loading from AppContext.BaseString. However, in the future we'd like all tools to be exclusively loaded from inside nuget libraries (that way we can tell at runtime if a tool changed by looking at its version number). When this happens, the variable name should be `notLoadedFromNuget`. Therefore I decided to name it `isUnsafe` and specify the meaning in a comment. 
If it's too vague though, I'll rename it to `isLoadedFromPath` :)